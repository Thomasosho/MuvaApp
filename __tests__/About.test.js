jest.useFakeTimers("modern");
import React from "react";
import renderer from "react-test-renderer";

import AboutScreen from "../src/screens/about/About";

// describe("<App />", () => {
//   it("has 1 child", () => {
//     const tree = renderer.create(<App />).toJSON();
//     expect(tree.children.length).toBe(1);
//   });
// });
it("renders correctly", () => {
  const tree = renderer.create(<AboutScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
