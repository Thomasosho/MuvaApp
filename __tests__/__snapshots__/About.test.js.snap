// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 16,
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#000",
        "fontSize": 14,
        "textAlign": "justify",
      }
    }
  >
    Muva is your creative friend! We are award-winning mobile app developers in Kenya with commitment to your satisfaction. Our team converts ideas to apps. We will help you design Android, iOS, SMS or USSD apps as well as web portals and a website to help you launch your app. We also handle integrations, that is, MPESA, CRB, Maps, Paypal as well as the smile you need on Monday morning! We are your go to people for all your website design in Nairobi, in Kenya and really, in the world.
  </Text>
</View>
`;
